external/mcpclient/turbo.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": ["//"],
+  "tasks": {
+    "build": {
+      "inputs": ["$TURBO_DEFAULT$", ".env*"],
+      "outputs": [".next/**", "!.next/cache/**"],
+      "cache": false,
+      "dependsOn": ["@microsoft/spark.common#build", "@microsoft/spark.ai#build"]
```

Comment:
since your not depending on spark.common anymore, it can be removed from this dependsOn Array